docker/Dockerfile.arm-unknown-linux-gnueabihf

```diff
@@ -13,8 +13,8 @@ RUN mkdir /usr/arm-linux-gnueabihf && \
     apt-get install --assume-yes --no-install-recommends curl xz-utils && \
     cd /usr/arm-linux-gnueabihf && \
     curl -L https://developer.arm.com/-/media/Files/downloads/gnu-a/8.3-2019.03/binrel/gcc-arm-8.3-2019.03-x86_64-arm-linux-gnueabihf.tar.xz | \
-    tar --strip-components 1 -xJ && \
-    apt-get purge --auto-remove -y curl xz-utils
+    tar --strip-components 1 -xJ
+    # packages not removed since they were originally installed in common.sh
```

Comment:
Maybe just remove the `apt-get install` line completely.

Reply:
Done in amended commit 843ad77bdb7b49cca78bd07da94ff053ed76fdf8.